HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponentsController+Helpers.swift

```diff
@@ -0,0 +1,806 @@
+        let shareInvoiceBottomView = ShareInvoiceBottomView(viewModel: shareInvoiceBottomViewModel, bottomSheetConfiguration: configurationProvider.bottomSheetConfiguration)
+        incrementOnboardingCountFor(paymentProvider: selectedPaymentProvider)
+        return shareInvoiceBottomView
+    }
+    
+    /**
+     Updates the selected payment provider with the given payment provider. This method is used when updating the payment provider from Payment Review Screen
+
+     - Parameters:
+       - paymentProvider: The new payment provider to be set.
+     */
+    public func updatedPaymentProvider(_ paymentProvider: GiniHealthAPILibrary.PaymentProvider) {
+        self.selectedPaymentProvider = PaymentProvider(healthPaymentProvider: paymentProvider)
+    }
+
+    /**
+     Opens the more information view controller by notifying the view delegate. This method is used when opening the More Information screen inside the bank selection bottom sheet that's presented in the Payment Review Screen.
+
+     This method triggers the delegate's action for displaying more information.
+     */
+    public func openMoreInformationViewController() {
+        didTapOnMoreInformation(documentId: documentId)
+    }
+    
+    // MARK: - Other helpers
+
+    private func incrementOnboardingCountFor(paymentProvider: PaymentProvider?) {
+        var onboardingCounts = OnboardingShareInvoiceScreenCount.load()
+        onboardingCounts.incrementPresentationCount(forProvider: paymentProvider?.name)
+    }
+
+    func setupObservers() {
+        NotificationCenter.default.addObserver(self, selector: #selector(paymentInfoDissapeared), name: .paymentInfoDissapeared, object: nil)
+    }
+    
+    @objc
+    private func paymentInfoDissapeared() {
+        if previousPresentedView == .bankPicker {
+            didTapOnBankPicker()
+        } else if previousPresentedView == .paymentComponent {
+            didTapOnPayButton()
+        }
+        previousPresentedView = nil
+    }
+    
+    /// Checks if the payment provider app can be opened based on the selected payment provider and GPC(Gini Pay Connect) support.
+    public func canOpenPaymentProviderApp() -> Bool {
+        if supportsGPC() {
+            if healthSelectedPaymentProvider?.appSchemeIOS.canOpenURLString() == true {
+                return true
+            }
+        }
+        return false
+    }
+
+
+    /// Checks if the selected payment provider supports the "Open With" feature on iOS.
+    public func supportsOpenWith() -> Bool {
+        if healthSelectedPaymentProvider?.openWithSupportedPlatforms.contains(.ios) == true {
+            return true
+        }
+        return false
+    }
+
+    /// Checks if the selected payment provider supports GPC(Gini Pay Connect) on iOS.
+    public func supportsGPC() -> Bool {
+        if healthSelectedPaymentProvider?.gpcSupportedPlatforms.contains(.ios) == true {
+            return true
+        }
+        return false
+    }
```

Comment:
[suggestion] we can directly return `healthSelectedPaymentProvider?.gpcSupportedPlatforms.contains(.ios) == true`  instead of using if. What do you say?